src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Model/AzureSqlElasticPoolModel.cs

```diff
@@ -92,5 +120,27 @@ public class AzureSqlElasticPoolModel
         /// Gets or sets the zone redundant option of the elastic pool.
         /// </summary>
         public bool? ZoneRedundant { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Sku of the elastic pool.
+        /// </summary>
+        public Sku Sku { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the per database settings of the elastic pool.
+        /// </summary>
+        public ElasticPoolPerDatabaseSettings PerDatabaseSettings { get; set; }
```

Comment:
Remove this since the data in here is already in MinCapacity  and MaxCapacity properties

Reply:
@jaredmoo The "PerDatabaseSettings" is used when applyUserInputToModel for create/update pool, and passed as part of Entity. I don't think it can be removed here.

Reply:
Why do we need both PerDatabaseSettings ad Min/MaxCapacity? We should not need both.

Reply:
@jaredmoo when return elastic pool response, I flattened them using DatabaseCapacityMin/Max now. Will not use perDatabaseSettings in the response, leave it as empty. But I keep PerDatabaseSettings property in ElasticPoolModel since use it when create request entity.

Also for Capacity, Family, flatten them